infra/terraform/modules/data_access/gh_webhook_handler/index.py

```diff
@@ -0,0 +1,91 @@
+import hmac
+from hashlib import sha1
+import json
+import os
+
+import boto3
+from botocore.exceptions import ClientError
+
+
+SNS_ARN = os.environ.get("SNS_ARN")
+STAGE = os.environ.get("STAGE")
+GH_HOOK_SECRET = os.environ.get("GH_HOOK_SECRET")
+
+sns = boto3.client("sns")
+
+
+class SNSTopicNotFoundError(Exception):
```

Comment:
Funny how I didn't include the `Error` suffix in the first place and then when I looked at the [Python documentation](https://docs.python.org/3/tutorial/errors.html#user-defined-exceptions) and it said:

>  Most exceptions are defined with names that end in “Error,” similar to the naming of the standard exceptions."

So I said, alright...I'll add the Error. Happy to remove if that's OK with The Python Way (TM) :P